pkg/controller/controller.go

```diff
@@ -0,0 +1,12 @@
+package controller
+
+import (
+	"genesis-currency-api/internal/service"
+	"github.com/gin-gonic/gin"
+)
+
+func RegisterAllRoutes(r *gin.Engine, cs *service.CurrencyService, us *service.UserService, es *service.EmailService) {
+	CurrencyRegisterRoutes(r, cs)
+	UserRegisterRoutes(r, us, es)
+	UtilRegisterRoutes(r, us, es)
```

Comment:
I don't know about naming conventions in Go, but in my opinion these functions must have a verb in the name